app/stacks/cumulus/resources/collections/WV01_Pan_L1B___1.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "WV01_Pan_L1B",
+  "version": "1",
+  "duplicateHandling": "replace",
+  "granuleId": ".*",
+  "granuleIdExtraction": "^(WV01_.+-P1BS-.+_P\\d{3}).+$",
+  "sampleFileName": "WV01_20071231044012_1020010001E9B500_07DEC31044012-P1BS-505051421030_01_P024-BROWSE.jpg",
+  "url_path": "{cmrMetadata.CollectionReference.ShortName}___{cmrMetadata.CollectionReference.Version}/{dateFormat(cmrMetadata.TemporalExtent.SingleDateTime, YYYY)}/{dateFormat(cmrMetadata.TemporalExtent.SingleDateTime, DDD)}/{cmrMetadata.GranuleUR}",
+  "meta": {
+    "preferredQueueBatchSize": 5
+  }
```

Comment:
You need a comma after the `}`.

I suspect you didn't try replacing the collection, otherwise you would have gotten an error:
```
cumulus collections replace --data app/stacks/cumulus/resources/collections/WV01_Pan_L1B___1.json
```